packages/cli/src/ErrorReporting.ts

```diff
@@ -39,7 +39,7 @@ export const initErrorHandling = async () => {
 
 	const seenErrors = new Set<string>();
 	addGlobalEventProcessor((event, { originalException }) => {
-		if (originalException instanceof NodeError && originalException.severity === 'warning')
+		if (originalException instanceof ExecutionBaseError && originalException.severity === 'warning')
```

Comment:
The team should be made aware that an error with `warning` severity will not reach Sentry. Maybe post in Slack after merging and update any Sentry docs we might have in Notion.